app1.py

```diff
@@ -4,6 +4,7 @@
 
 df = pd.read_csv('./final_0427.csv')
 photos = pd.read_csv('./photos_big.csv')

```

Comment:
I believe it should be `photos = pd.read_csv('./extraneous_datasets/photos_big.csv')`